README.md

```diff
@@ -13,7 +13,7 @@ This is a list of clouds that we test on a nightly basis against our `scs-compat
 | [Wavestack](https://www.noris.de/wavestack-cloud/)                                                             | Public cloud for customers                        | noris network AG/Wavecon GmbH | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-wavestack-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-wavestack-v3.yml) |                               [HM](https://health.wavestack1.sovereignit.cloud:3000/)                                    |
 | [REGIO.cloud](https://regio.digital)                                                                           | Public cloud for customers                        | OSISM GmbH                    | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-regio-a-v4.yml?label=v4)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-regio-a-v4.yml)   |   broken <!--[HM](https://apimon.services.regio.digital/public-dashboards/17cf094a47404398a5b8e35a4a3968d4?orgId=1&refresh=5m)-->      |
-| [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | [artcodix UG](https://artcodix.com/) | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
+| [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | [artcodix GmbH](https://portal.cnds.io/) | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
```

Comment:
I would prefer to remove the artcodix GmbH link (like for all of the other companies) or to link the company website itself. CNDS is already linked to cnds.io.

Reply:
Yes, I was about to suggest the same.
Imho the name should link to the site that displays the offering and that should be the only link - just like with all other entries in the table.

Reply:
i guess you are right. either remove it or change it back. ill commit an update

Reply:
done